ios/Barometer.m

```diff
@@ -37,14 +37,9 @@ + (BOOL)requiresMainQueueSetup
 - (void) isAvailableWithResolver:(RCTPromiseResolveBlock) resolve
                         rejecter:(RCTPromiseRejectBlock) reject {
 
-    if (![CMAltimeter isRelativeAltitudeAvailable]) 
+    if ([CMAltimeter isRelativeAltitudeAvailable]) 
     {
-        if([CMPedometer authorizationStatus] == CMAuthorizationStatusAuthorized)
-        {
-            resolve(@YES);
-        } else {
-            reject(@"-1", @"Barometer is not authorized", nil);
-        }
+        resolve(@YES);
```

Comment:
Like this change 👍 We do not need to check an auth status for the Altimeter right?

Reply:
For me the auth status never returned `CMAuthorizationStatusAuthorized`, which is why I removed the check, and the barometer finally started working. So the barometer doesn't need any authorization it seems.

Reply:
Not even for the Altimeter. We had the pedimeter before

Reply:
I'm sorry. We had CMPedometer there before, which seems to be the wrong sensor. Did you try to send the Auth request to the altimeter instead?

Reply:
@DanielMSchmidt, I tried with `CMAltimeter authorizationStatus` now, which returns `false`. However, I'm still able to read the barometer, so I'm guessing the `isRelativeAltitudeAvailable` check alone is enough for the barometer.

Reply:
Sounds good to me 👍